### ISSUE TITLE
Update the entire environment.yml file (and Docker image) to use latest versions of all R packages

### ISSUE THREAD

**Issue by mmeier93** [lifebit-ai/prs#5]
I used the existing environment.yml file when developing this pipeline (October 2020). I added the following packages:

```
  - conda-forge::r-tidyverse=1.3.0 
  - conda-forge::r-snakecase=0.11.0
  - conda-forge::r-sm=2.2_5.6
```

However, I am currently getting warnings when I load all packages in my R script.

For example, `plot_prs_vs_density.R`:

```
suppressPackageStartupMessages({
  library(tidyverse)
  library(data.table)
  library(ggplot2)
  library(snakecase)
  library(sm)
})
```

Will produce the following warnings:
```
Warning messages:
1: package ‘tidyverse’ was built under R version 3.6.3 
2: package ‘tibble’ was built under R version 3.6.3 
3: package ‘tidyr’ was built under R version 3.6.3 
4: package ‘readr’ was built under R version 3.6.3 
5: package ‘purrr’ was built under R version 3.6.3 
6: package ‘dplyr’ was built under R version 3.6.3 
7: package ‘stringr’ was built under R version 3.6.3 
8: package ‘forcats’ was built under R version 3.6.3 
9: package ‘snakecase’ was built under R version 3.6.3 
```

In addition, when loading sm **locally**, I would get this warning:
```
/Library/Developer/CommandLineTools/usr/bin/objdump: error: '/Users/mmeier/anaconda3/envs/prs/lib/R/library/tcltk/libs//tcltk.so': No such file or directory
Package 'sm', version 2.2-5.6: type help(sm) for summary information
Warning message:
In system2("/usr/bin/otool", c("-L", shQuote(DSO)), stdout = TRUE) :
  running command ''/usr/bin/otool' -L '/Users/mmeier/anaconda3/envs/prs/lib/R/library/tcltk/libs//tcltk.so'' had status 1
```

I am using R version 3.6.1 in the environment.yml. Attempts to change it to 3.6.3 or more recent resulted in many errors so things have been left as is for now.

In addition, the pipeline is not using the most recent version of PRSice.

**=> Aim: rebuild the environment.yml file from "scratch":**
- Check which packages are actually needed (not sure some of the original packages are needed)
- Use more recent versions for all packages
- In particular, make sure no error is produced by the `sm` package
- Use more recent version of PRSice

**Comment by mmeier93** [comment]
In addition, adding the `library(optparse)` to the R script `plot_prs_density.R`, the following error message would appear (although the plot is produced correctly):

![Screenshot 2020-10-12 at 18 09 36](https://user-images.githubusercontent.com/33165265/95774206-d8c0e200-0cb7-11eb-902b-64d1fbf3d7a1.png)

**Comment by mmeier93** [comment]
Attempts on 15/10/20:

Using this environment.yml:
```
# You can use this file to create a conda environment for this pipeline:
#   conda env create -f environment.yml
name: prs
channels:
  - conda-forge
  - bioconda
  - defaults
dependencies:
  # conda-forge channel packages first
  - conda-forge::r-base=4.0.3
  - conda-forge::r-tidyverse=1.3.0
  - conda-forge::r-data.table=1.13.0
  - conda-forge::r-optparse=1.6.6
  - conda-forge::r-rmarkdown=2.4
  - conda-forge::r-snakecase=0.11.0
  - conda-forge::r-sm=2.2_5.6 
  - conda-forge::r-r.utils=2.10.1
  - conda-forge::r-dt=0.16
  - conda-forge::r-htmltools=0.5.0
  - conda-forge::r-kableextra=1.2.1

  # bioconda channel packages second
  # defaults channel packages third
  - fonts-anaconda
```
In addition, I attempted to update the PRSice package within the Docker file from 2.2.10 to 2.3.3.

The env works well but when attempting to run the pipeline but when attempting to run the pipeline I get this new error on the process running PRSice:

![Screenshot 2020-10-15 at 10 37 36](https://user-images.githubusercontent.com/33165265/96106206-d6869f80-0ed2-11eb-8547-f191ed6f1016.png)

For now, have reverted back to the old versions of .yml and Dockerfile.

**Comment by mmeier93** [comment]
Corrected during Sprint #115 PR.
